### PR TITLE
November newsletter blog

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

November newsletter blog

Rendered at https://20241101-november-2024-newsl.ddev-com-front-end.pages.dev/blog/ddev-nov-2024-newsletter/

### CODE REVIEW

File: src/content/blog/ddev-nov-2024-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+**Contributor Trainings From October (Recordings and Blogs)**
+
+* [Supporting people using `ddev debug test` and other tools](ddev-debug-test-contributor-training.md)
+* [Using Tmate to Debug GitHub Workflows**](tmate-github-actions-contributor-training.md)
+
+**DDEV Notes from around the web:**
+
+- **[DDEV Release v1.23.5 Announcement](https://ddev.com/blog/release-v1235-auto-port-assignment). Lots of goodies in this latest release, including automatic port assignment and automatic time zone detection.
+- **[Mauricio Dinarte (@dinarcon)'s DrupalCon presentation on DDEV](https://www.youtube.com/watch?v=nPJC7BbiGNw&list=PLpeDXSh4nHjQOfQV-BUgoxHXlr4tHlhPO&index=16&pp=iAQB)**
```
Comment (rpkoller) [inline src/content/blog/ddev-nov-2024-newsletter.md:46]:
```suggestion
- **[Mauricio Dinarte's (@dinarcon) DrupalCon presentation on DDEV](https://www.youtube.com/watch?v=nPJC7BbiGNw&list=PLpeDXSh4nHjQOfQV-BUgoxHXlr4tHlhPO&index=16&pp=iAQB)**
```

File: src/content/blog/ddev-nov-2024-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+featureImage:
+  src: /img/blog/2024/11/palisade-november-bookcliffs.jpg
+  alt: Palisade, Colorado, USA, November 1, 2024
+categories:
+  - Community
+---
+
+**Happy November from DDEV**! It's a beautiful fall here, but things are cooling down. We live in an agricultural area, so the first freeze is always an event.
+
+**It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable (and fully privileged) co-maintainer:
```
Comment (rpkoller) [inline src/content/blog/ddev-nov-2024-newsletter.md:16]:
```suggestion
**It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable co-maintainer who has full privileges for the DDEV repository.
```

the suggestion is not perfect at all wording wise. but i just wanted to add some context to "fully privileged". cuz without the context i was uncertain what privileged was actually referring to and what about. about my suggestion, i was uncertain about the scope. is he fully privileged for the ddev repo or the org in general. that might need some readjustment same as the wording in general has room to improve.

**Review by rpkoller — COMMENTED** [review]
added suggestions about two minor details. in general the article reads well

File: src/content/blog/ddev-nov-2024-newsletter.md (hunk 2 above)
Comment (rfay) [reply src/content/blog/ddev-nov-2024-newsletter.md:16]:
```suggestion
**It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable co-maintainer who has full privileges on all things DDEV.
```

File: src/content/blog/ddev-nov-2024-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+> The `ddev launch :port` feature has been really useful. It's hard to imagine that DDEV didn't have it before.
+
+> I recently added automatic timezone detection in the web container. It seems like a small thing, but it has a big impact on simplicity and is very convenient for developers.
+
+> Finally, the upcoming `ddev cd` feature is something I’m excited about. I used to have a custom Bash script for this, but now it will be built into DDEV, with autocomplete.
+
+**Advisory Group Meeting Wednesday**: All are invited to the DDEV Advisory Group meeting Wednesday, November 6, 9am US MT, 5PM CET. [Meetup link](https://www.meetup.com/ddev-events/events/303197392/?eventOrigin=group_events_list) or [Zoom link](https://us02web.zoom.us/j/7315692237?pwd=RHR6NUkwb0g5WXIzS2NOcXRucCthZz09&omn=89153105842), passcode "12345" if you need it.
+
+**Contributor Trainings From October (Recordings and Blogs)**
```
Comment (stasadev) [inline src/content/blog/ddev-nov-2024-newsletter.md:38]:
```suggestion
**Contributor Trainings From October (Recordings and Blogs):**
```

File: src/content/blog/ddev-nov-2024-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+* [Supporting people using `ddev debug test` and other tools](ddev-debug-test-contributor-training.md)
+* [Using Tmate to Debug GitHub Workflows**](tmate-github-actions-contributor-training.md)
```
Comment (stasadev) [inline src/content/blog/ddev-nov-2024-newsletter.md:41]:
```suggestion
* **[Supporting people using `ddev debug test` and other tools](ddev-debug-test-contributor-training.md)**
* **[Using Tmate to Debug GitHub Workflows](tmate-github-actions-contributor-training.md)**
```

File: src/content/blog/ddev-nov-2024-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+**DDEV Notes from around the web:**
+
+- **[DDEV Release v1.23.5 Announcement](https://ddev.com/blog/release-v1235-auto-port-assignment). Lots of goodies in this latest release, including automatic port assignment and automatic time zone detection.
```
Comment (stasadev) [inline src/content/blog/ddev-nov-2024-newsletter.md:45]:
```suggestion
- **[DDEV Release v1.23.5 Announcement](https://ddev.com/blog/release-v1235-auto-port-assignment).** Lots of goodies in this latest release, including automatic port assignment and automatic time zone detection.
```

**Review by stasadev — APPROVED** [review]

**Pull request merged**